README.md

```diff
@@ -28,28 +28,9 @@ An ANSI color-code highlighting lexer for Pygments.
    ```python
    from pygments_ansi_color import color_tokens
 
-   # Note: You can use different background colors for improved readability.
-   fg_colors = bg_colors = {
-       'Black': '#000000',
-       'Red': '#EF2929',
-       'Green': '#8AE234',
-       'Yellow': '#FCE94F',
-       'Blue': '#3465A4',
-       'Magenta': '#c509c5',
-       'Cyan': '#34E2E2',
-       'White': '#F5F5F5',
-       'BrightBlack': '#676767',
-       'BrightRed': '#FF6D67',
-       'BrightGreen': '#5FF967',
-       'BrightYellow': '#FEFB67',
-       'BrightBlue': '#6871FF',
-       'BrightMagenta': '#FF76FF',
-       'BrightCyan': '#5FFDFF',
-       'BrightWhite': '#FEFFFF',
-   }
    class MyStyle(pygments.styles.xcode.XcodeStyle):
        styles = dict(pygments.styles.xcode.XcodeStyle.styles)
-       styles.update(color_tokens(fg_colors, bg_colors))
+       styles.update(color_tokens())
```

Comment:
I think it would still be good to include how to customize the token colors somewhere in the README since it is one of the main usage questions people are likely to have about this library.

Adding these as defaults is great but could we still include instructions somewhere? Can be e.g. at the bottom with the other optional instructions.

Reply:
Oh yes you're right.

I just develop the README to add code example and explanation on how to set your own color theme.